src/querystringsafe_base64.py

```diff
@@ -37,7 +37,7 @@ def encode(to_encode):
         string - like base64, except characters ['+', '/', '='] are
         replaced with ['-', '_', '.'] consequently
     """
-    return urlsafe_b64encode(bytes(to_encode)).replace('=', '.')
+    return urlsafe_b64encode(('%s' % to_encode).encode()).decode().replace('=', '.')
```

Comment:
No more readable solution? Is the param really an `str`, or only `bytes` on Python 3?